tests/components/climate/test_demo.py

```diff
@@ -224,10 +224,10 @@ def test_set_hold_mode_away(self):
 
     def test_set_hold_mode_none(self):
         """Test setting the hold mode off/false."""
-        climate.set_hold_mode(self.hass, None, ENTITY_ECOBEE)
+        climate.set_hold_mode(self.hass, 'off', ENTITY_ECOBEE)
         self.hass.block_till_done()
         state = self.hass.states.get(ENTITY_ECOBEE)
-        self.assertEqual(None, state.attributes.get('hold_mode'))
+        self.assertEqual('off', state.attributes.get('hold_mode'))
```

Comment:
@baloob I am a bit worried about this test as I couldn't figure out if None was supposed to be an allowed value.

Reply:
`None` means that the attribute does not exist in `state.attributes`